### ISSUE TITLE
3.3.3 regression - error from @extend inside @include at root level

### ISSUE THREAD

**Issue by bwbuchanan** [sass/libsass#1915]
I believe this regression may have been introduced in libsass 3.3.3 by the fix for #1651 .

# Minimal test-case

```
@mixin wrapper() {
  .wrapped {
    @content;
  }
}

%ext {
  background: #000;
}

@include wrapper() {
  @extend %ext;
}

```

# Expected output

Ruby sass 3.4.21 and libsass 3.3.2 produce this correct output:

```
.wrapped {
  background: #000;
}

```

# libsass 3.3.3 behavior

sassc with libsass 3.3.3 instead emits this error:

```
Error: Extend directives may only be used within rules.
        on line 12 of test.scss
>>   @extend %ext;
   --^

```

**Comment by xzyfer** [comment]
Thanks for the report @bwbuchanan. I've confirmed this is a regression in 3.3.3.

**Comment by xzyfer** [comment]
Spec added https://github.com/sass/sass-spec/pull/739

**Comment by xzyfer** [comment]
Fixed by #1953.
